cli/src/main/java/hudson/cli/CLI.java

```diff
@@ -40,15 +43,16 @@
 import java.security.KeyPair;
 import java.security.SecureRandom;
 import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
 import java.util.Locale;
 import java.util.Properties;
 import java.util.logging.Handler;
 import java.util.logging.Level;
 import java.util.logging.Logger;
-import static java.util.logging.Level.*;
-import org.apache.commons.io.FileUtils;
```

Comment:
Moving FileUtils and StringUtils earlier in the file seems like it doesn't change much and may cause conflicts with other pull requests.  Could those be left in their current location?